lang/en.json

```diff
@@ -19,7 +19,7 @@
   "content.contributors": "Contributors",
   "content.home.columned-hero.title": "<span class=\"supercharge__gradient\">Supercharge</span>your Minecraft gameplay.",
   "content.home.columned-hero.subtitle": "A simple Minecraft modpack focusing on performance and graphics enhancements.",
-  "content.home.graph.subtitle": "Fabulously Optimized is <span class=\"smaller__gradient\">5x Quicker</span> than vanilla Minecraft.<br />",
+  "content.home.graph.subtitle": "Fabulously Optimized is up to <span class=\"smaller__gradient\">5× Quicker</span> than vanilla Minecraft.<br />",
   "content.home.graph.fps": "FPS",
   "content.home.graph.small": "Tested on AMD Ryzen 5 2600 (12) &commat; 3.400GH and NVIDIA GeForce GTX 1060 3GB running vanilla Minecraft 1.20.1, a popular performance-focused game client for Minecraft 1.20.1 and Fabulously Optimized 5.4.1 (singleplayer).",
```

Comment:
Add something like. "Performance improvement might vary"

Reply:
well, "up to" already implies that
and the details below describe where it was tested

Reply:
Just to make it clearer